04/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+.container {
+	margin: 0 auto;
+	max-width: 1024px;
+}
+.main-nav {
+	background: #a1a1a1;
+	width: 100%;
+}
+
+.main-nav__btn {
+	display: block;
+	margin-left: auto;
+	padding: 0 30px;
+	line-height: 40px;
+	background: none;
+	border: none;
+	cursor: pointer;
+}
+
+.main-nav__list,
+.dropdown {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	text-decoration: none;
+	list-style: none;
+	display: none;
+}
+
+/* .main-nav__item. {
+} */
+
+.main-nav__link,
+.dropdown__link {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	color: #fff;
+	text-decoration: none;
+	min-width: 120px;
+	height: 50px;
+	background: #a1a1a1;
+}
+
+.main-nav__link:hover,
+.dropdown__link:hover {
+	color: aqua;
+}
+.dropdown__list {
+	display: none;
+	position: absolute;
+	text-align: center;
+	top: 50px;
+	background-color: #f9f9f9;
+	min-width: 160px;
+	box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+	z-index: 1;
+}
+
+.dropdown:hover > .dropdown__list {
+	display: block;
+}
```

Comment:
Niestety w wersji mobilnej to nie działa - trzeba użyć `focus` lub zmienić podejście na `input type="checkbox"`